.circleci/config.yml

```diff
@@ -421,6 +421,66 @@ jobs:
       - slack/notify-on-failure:
           only_for_branches: master,production
 
+  deploy-shared:
+    parameters:
+      serviceName:
+        type: string
+    executor: python-executor
+    working_directory: ~/back
+    steps:
+      # Attach workspace from build
+      - attach_workspace:
+          at: ~/
+      # install gcloud orb
+      - gcp-cli/install
+      - run:
+          name: Create version.txt
+          command: |
+            echo "CircleCI build number:${CIRCLE_BUILD_NUM}
+              Branch: ${CIRCLE_BRANCH}
+              Git hash: ${CIRCLE_SHA1}" > version.txt
+      - run:
+          # GAE only installs dependencies from requirements.txt
+          name: Extend dependencies to be installed in deploy-environment
+          command: cat requirements-deploy.txt >> requirements.txt
+      - run:
+          name: Generate .env file
+          command: |
+            echo "
+              MYSQL_USER=${DB_NAME}
+              MYSQL_PASSWORD=${DB_PASS}
```

Comment:
Let's create or use existing SELECT only db user

Reply:
@HaGuesto reminder to send me the values for staging/demo such that I can set them in the CircleCI contexts